mlflow/utils/gorilla.py

```diff
@@ -0,0 +1,898 @@
+#                     __ __ __
```

Comment:
We inline gorilla to make the change described here: https://github.com/mlflow/mlflow/pull/3582/files#r512233266